src/app/SharpRaven/Data/Breadcrumb.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+
+using SharpRaven.Serialization;
+
+namespace SharpRaven.Data {
+    /// <summary>
+    /// Breadcrumb trail.
+    /// </summary>
+    public class Breadcrumb {
+        private readonly DateTime timestamp;
+
+        public Breadcrumb() 
+        {
+            Category = "log";
```

Comment:
Within our other specs we explicitly require crumbs to define a category (rather than providing a default). While not particularly an issue, I would suggest the same here. Anything that automatically creates breadcrumbs is also of providing its own category, so forcing users to create one also helps educate them on the control/feature.